common/src/main/java/nearsoft/skytouch/common/ProductJSONSerializer.java

```diff
@@ -0,0 +1,59 @@
+package nearsoft.skytouch.common;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import nearsoft.skytouch.common.model.Product;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+
+public class ProductJSONSerializer {
```

Comment:
This class is fine but in my opinion it is not needed at all. The serializer/deserializer itself is the ObjectMapper, so you could use it directly where you need it. 

Following this approach you should create a serializer and deserializer method for every entity... and you are creating a pair for a single object and a pair for a list... imagine this with 20 entities ... is it useful?

Reply:
No for 20 entities is not useful, but i think in this case make more clearer the code in the classes in which i'm using, do you recommend to change it to use directly the ObjectMapper?

Reply:
No... maybe when the entities quantity increases you could use this class but with Generics